File: app/src/main/java/com/adisa/diningplus/utils/DateFormatProvider.java
package com.adisa.diningplus.utils;

import java.text.SimpleDateFormat;
import java.util.Locale;

public class DateFormatProvider {
    public static final SimpleDateFormat full = new SimpleDateFormat("MMMMM, dd yyyy HH:mm:ss", Locale.US);
    public static final SimpleDateFormat date = new SimpleDateFormat("yyyy-MM-dd", Locale.US);
    public static final SimpleDateFormat time = new SimpleDateFormat("HH:mm:ss", Locale.US);
    public static final SimpleDateFormat hour = new SimpleDateFormat("h:mm a", Locale.US);
    public static final SimpleDateFormat hour24 = new SimpleDateFormat("HH:mm", Locale.US);
}

File: app/src/main/java/com/adisa/diningplus/utils/DateFormatProvider.kt
package com.adisa.diningplus.utils

import java.text.SimpleDateFormat
import java.util.*

object DateFormatProvider {
    val full = SimpleDateFormat("MMMMM, dd yyyy HH:mm:ss", Locale.US)
    val date = SimpleDateFormat("yyyy-MM-dd", Locale.US)
    val dateFull = SimpleDateFormat("MMMM d", Locale.US)
    val time = SimpleDateFormat("HH:mm:ss", Locale.US)
    val hour = SimpleDateFormat("h:mm a", Locale.US)
    val hour24 = SimpleDateFormat("HH:mm", Locale.US)
}